website/blog/2022/02/21/nacos.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "How to Use Apache APISIX Realizes Service Discovery Based on Nacos"
+authors:
+  - name: "Zhihuang Lin"
+    title: "Author"
+    url: "https://github.com/oil-oil"
+    image_url: "https://avatars.githubusercontent.com/u/57465570?v=4"
+  - name: "Fei Han"
+    title: "Technical Writer"
+    url: "https://github.com/hf400159"
+    image_url: "https://avatars.githubusercontent.com/u/97138894?v=4"
+keywords: 
+- Apache APISIX
+- Nacos
+- Service Discovery
+- Service Registry
+- Ecosystem
+description: This article introduces the basic concepts of Apache APISIX and Nacos and Service Registry, and shows you the specific operation of Apache APISIX to realize service discovery based on Nacos.
+tags: [Technology,Ecosystem,Service Discovery]
+---
+
+> This article introduces the basic concepts of Apache APISIX and Nacos and Service Registry, and shows you the specific operation of Apache APISIX to realize service discovery based on Nacos.
+
+<!--truncate-->
+
+## Background Information
+
+Nacos is an easy-to-use, open source platform for dynamic service discovery, service configuration and service management. It provides a set of simple and useful features enabling you to realize dynamic service discovery, service configuration, service metadata and traffic management. Nacos makes it easier and faster to construct, deliver and manage your microservices platform. It is the infrastructure that supports a service-centered modern application architecture with a microservices or cloud-native approach.
+
+Service Registry is the core component of service management, similar to the role of directory service, and one of the most basic facilities in the microservices architecture. It is mainly used to store service information, such as service provider URL, routing information, and so on. The service registry is implemented by mapping complex service-side information to simple and understandable information for the client.
+
+The core functions of the Service Registry are as follows:
+
+- Service registration: Service providers register with the Service Registration Center.
+- Service discovery: Service consumers can find the call routing information of service providers through the registry.
```

Comment:
Need to add bold to some phrases
![image](https://user-images.githubusercontent.com/36651058/155108695-51b22781-9b19-4261-a73e-d027ef487bd6.png)
```suggestion
- Service registration: **Service providers** register with the **Service Registration Center**.
- Service discovery: **Service consumers** can find the call routing information of service providers through the registry.
```